addons/refuel/functions/fnc_startNozzleInHandsPFH.sqf

```diff
@@ -38,7 +38,7 @@ TRACE_2("start",_unit,_nozzle);
         alive _unit
         && {"" isEqualTo currentWeapon _unit || {_unit call EFUNC(common,isSwimming)}}
         && {[_unit, objNull, [INTERACT_EXCEPTIONS, "notOnMap"]] call EFUNC(common,canInteractWith)}
-        && {"unconscious" isNotEqualTo toLower animationState _unit}
+        && {"unconscious" isNotEqualTo toLowerANSI animationState _unit}
```

Comment:
`_unit call EFUNC(common,isAwake)` can replace

`alive _unit`, `&& {"unconscious" isNotEqualTo toLower animationState _unit}` and `&& {!(_unit getVariable ["ACE_isUnconscious", false])}`

I know it's not part of the PR's goal, so I understand if you decide not to include the suggestion.